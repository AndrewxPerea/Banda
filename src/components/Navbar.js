import React from 'react';
import { Navbar, Nav, Container } from 'react-bootstrap';
import { LinkContainer } from 'react-router-bootstrap';

function MyNavbar() {
  return (
    <Navbar bg="dark" variant="dark" expand="lg" sticky="top">
      <Container>
        <Navbar.Brand href="/">
          <img
            src="/logo.png" 
            height="40"  
            className="d-inline-block align-top"
            alt="Siete Octavos Logo"
          />
        </Navbar.Brand>

        <Navbar.Toggle aria-controls="basic-navbar-nav" />
        <Navbar.Collapse id="basic-navbar-nav">
          <Nav className="ml-auto">
            <LinkContainer to="/">
              <Nav.Link>Inicio</Nav.Link>
            </LinkContainer>
            <LinkContainer to="/about">
              <Nav.Link>Sobre Nosotros</Nav.Link>
            </LinkContainer>
            <LinkContainer to="/music">
              <Nav.Link>Música</Nav.Link>
            </LinkContainer>
            <LinkContainer to="/gallery">
              <Nav.Link>Galería</Nav.Link>
            </LinkContainer>
            <LinkContainer to="/contact">
              <Nav.Link>Contacto</Nav.Link>
            </LinkContainer>
          </Nav>
        </Navbar.Collapse>
      </Container>
    </Navbar>
  );
}

export default MyNavbar;
