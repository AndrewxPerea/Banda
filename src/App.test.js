import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom'; // Importa el Router
import Navbar from './components/Navbar';

test('renders the logo and navigation links', () => {
  // Envuelve tu componente en un Router
  const { container } = render(
    <BrowserRouter>
      <Navbar />
    </BrowserRouter>
  );
  expect(container).toMatchSnapshot();
});
