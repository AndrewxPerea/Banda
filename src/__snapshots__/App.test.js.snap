// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the logo and navigation links 1`] = `
<div>
  <nav
    class="navbar navbar-expand-lg navbar-dark bg-dark sticky-top"
  >
    <div
      class="container"
    >
      <a
        class="navbar-brand"
        href="/"
      >
        <img
          alt="Siete Octavos Logo"
          class="d-inline-block align-top"
          height="40"
          src="/logo.png"
        />
      </a>
      <button
        aria-controls="basic-navbar-nav"
        aria-label="Toggle navigation"
        class="navbar-toggler collapsed"
        type="button"
      >
        <span
          class="navbar-toggler-icon"
        />
      </button>
      <div
        class="navbar-collapse collapse"
        id="basic-navbar-nav"
      >
        <div
          class="ml-auto navbar-nav"
        >
          <a
            class="active nav-link"
            data-rr-ui-event-key="/"
            href="/"
          >
            Inicio
          </a>
          <a
            class="nav-link"
            data-rr-ui-event-key="/about"
            href="/about"
          >
            Sobre Nosotros
          </a>
          <a
            class="nav-link"
            data-rr-ui-event-key="/music"
            href="/music"
          >
            Música
          </a>
          <a
            class="nav-link"
            data-rr-ui-event-key="/gallery"
            href="/gallery"
          >
            Galería
          </a>
          <a
            class="nav-link"
            data-rr-ui-event-key="/contact"
            href="/contact"
          >
            Contacto
          </a>
        </div>
      </div>
    </div>
  </nav>
</div>
`;
